### ISSUE TITLE
Make * work for all files in directory

### ISSUE THREAD

**Issue by VitaliPom** [lunixbochs/ActualVim#172]
Hi, 
I'm trying to make <*> work for the whole project, I write html with css a lot. 
Is it possible to toggle the search menu of vimgrep somehow? For me meanwhile it doesn't work.
Vitali

**Comment by hakunin** [comment]
Yeah vimgrep is a must-have and now its not working at all for me.

**Comment by VitaliPom** [comment]
Hi @hakunin, I upgraded to some sort of gvim. Gvim at work and Mac vim at home, I indeed recommend you, really really check those out. With the correct bindings they have much of functionality others don't have and plus wherever you will go you will have your preconfiguration. They are really very useful and very handy.

**Comment by lunixbochs** [comment]
Anyone know what api the search menu of vimgrep uses?
